.changes/next-release/bugfix-AWSSDKforJavav2-8565660.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "bugfix",
+    "category": "AWS SDK for Java v2",
+    "contributor": "",
+    "description": "Fix expiration in past warning from ProfileFileRefresher."
```

Comment:
nit: Since ProfileFileRefresher is an internal implementation detail, I wouldn't mention in user-facing release notes. Maybe something like  "Fix expiration in past warning during credential profile loading"